pages/cloud/storage/highperf-limitations/guide.fr-ca.md

```diff
@@ -24,6 +24,10 @@ Illimité
 
 1 Gbps / connexion
 
+### Nombre maximum de requetes par seconde en écriture sur un bucket
```

Comment:
Please fix this typo: requetes -> requêtes

Reply:
done
thx